introduction.md

```diff
@@ -1,7 +1,2 @@
 # Welcome to Alan Repo!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?